feat: simplify readme. (#3429)
timescale/docs: about/release-notes.md

---

> title: Timescale release notes and future plans

Reviewer: we don't include any future plans, not sure why we say this.

Also, these are Timescale release notes for self-hosted products.

Author: Removed the future plans comment. I'd prefer not to specify self-hosted as we use all these products in Timescale Cloud.

excerpt: New features and fixes are released regularly in all Timescale products
keywords: [TimescaleDB, Timescale Cloud, TimescaleDB Toolkit, pgai, pgvectorscale, pgspot, live-migration, upgrades, updates, releases]
---

# Timescale release notes

> For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 

Reviewer: The changelog is about Timescale Cloud specifically. Let's make that clear here and in the changelog.

For example, a TimescaleDB release is included in the release notes on Github as soon as it it released but it's not published in the changelog until it is available on Timescale Cloud.

Author: As before. Don't want to give the impression that these products are self-hosted only.

> For information about major release for all Timescale product, see the [Changelog][changelog]. For product release 

Reviewer: ```suggestion
For information about new updates and improvement to Timescale Cloud, see the [Changelog][changelog]. For product release 
```

> notes, see:

Reviewer: ```suggestion
notes for our self-hosted / downloadable products, see:
```

> * [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data.

Reviewer: ```suggestion
* [TimescaleDB](https://github.com/timescale/timescaledb/releases) -  an open-source database that makes SQL scalable for time-series data, packaged as a PostgreSQL extension.
```

> * [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - ease all things analytics when using TimescaleDB.

Reviewer: ```suggestion
* [TimescaleDB Toolkit](https://github.com/timescale/timescaledb-toolkit/releases) - additional functions to ease all things analytics when using TimescaleDB.
```

* [pgai](https://github.com/timescale/pgai/releases) - brings AI workflows to your PostgreSQL database.

> * [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications.

Reviewer: ```suggestion
* [pgvectorscale](https://github.com/timescale/pgvectorscale/releases/tag/0.2.0) -  higher performance embedding search and cost-efficient storage for AI applications on PostgreSQL.
```

* [pgspot](https://github.com/timescale/pgspot/releases) - spot vulnerabilities in PostgreSQL extension scripts.

> * [live-migration](https://hub.docker.com/r/timescale/live-migration/tags) - a Docker image to migrate data to a Timescale Cloud service.

Reviewer: These are not release notes. We should probably have release notes for live-migration somewhere. @VineethReddy02 are we keeping release notes for this project in any public location?
